Resources/Storage/StorageBase.py

```diff
@@ -229,7 +229,7 @@ def getCurrentURL( self, fileName ):
     if not result['OK']:
       return result
     cwdUrl = result['Value']
-    fullUrl = '%s/%s' % ( cwdUrl, fileName )
+    fullUrl = '%s%s' % ( cwdUrl, fileName )
```

Comment:
Yes, totally agree, and this is the plan. However:
* changing the logic in FTS is much heavier
* the fact remains: getCurrentURL is wrong as such, there should not be any double slash. In fact, I believe we should remove this, since it is not used anywhere. But not now, now we need to release :-)